src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -896,6 +896,9 @@ type SynExpr =
     /// Debug points arising from computation expressions
     | DebugPoint of debugPoint: DebugPointAtLeafExpr * isControlFlow: bool * innerExpr: SynExpr
 
+    /// F# syntax: f?x
+    | Dynamic of funcExpr: SynExpr * qmark: range * argExpr: SynExpr * range: range
```

Comment:
Could you update changelog.md please with information about this.  Also check for any previous changes you've made to public API surface area which should be mentioned in there, thanks